.github/workflows/build_and_test_pr.yml

```diff
@@ -101,8 +101,7 @@ jobs:
       - name: Install pip and Clang-Tidy
```

Comment:
@k0suke-murakami Could you add apt-transport-https in name?

Reply:
Thanks, here is the commit https://github.com/tier4/AutowareArchitectureProposal.iv/pull/713/commits/10fccf7cfcd77de61197e63db7659aeb94ff4741